### DISCUSSION TITLE
Can a 'search' box be added to the dashboard?

### DISCUSSION THREAD

**Discussion by gds03** [gitextensions/gitextensions#10302] [General]
HI everyone.

I have more than 50 repos, and is so hard to find any repo there. I guess I'm not the only one with this need.

Can a new search box be added to search for repo names faster?

**Comment by pmiossec** [comment]
#9380 (and #9536 )

**Reply by gds03** [reply]
when is there a release with this changes?

**Reply by pmiossec** [reply]
It's an opensource free time project so no defined date.
But we could expect not too long (a matter of weeks).
We are trying to fix the last bugs and there is a wish in the team to release it as soon as possible (but some things need to be finished and guarantee the quality of the release)

Discussion is here: https://github.com/gitextensions/gitextensions/discussions/9713

**Reply by gds03** [reply]
amazing stuff. 

Would give value to a lot of users.

Thanks @pmiossec

**Reply by gerhardol** [reply]
try the beta for now
